pkg/services/cosmosdb/cosmosdb-deprovision.go

```diff
@@ -0,0 +1,41 @@
+package cosmosdb
+
+import (
+	"context"
+
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+)
+
+func (c *cosmosManager) GetDeprovisioner(
+	service.Plan,
+) (service.Deprovisioner, error) {
+	return service.NewDeprovisioner(
+		service.NewDeprovisioningStep("deleteARMDeployment", c.deleteARMDeployment),
+		service.NewDeprovisioningStep(
+			"deleteCosmosDBServer",
+			c.deleteCosmosDBServer,
+		),
+	)
+}
+
+func (c *cosmosManager) deleteARMDeployment(
+	_ context.Context,
+	instance service.Instance,
+) (service.InstanceDetails, service.SecureInstanceDetails, error) {
+	err := deleteARMDeployment(c.armDeployer, instance)
+	if err != nil {
+		return nil, nil, err
+	}
+	return instance.Details, instance.SecureDetails, nil
+}
+
+func (c *cosmosManager) deleteCosmosDBServer(
```

Comment:
Cosmos "server?" Copy/pasta, probably. Or a relic of the first iteration of this module. Can we change this to "account?"

Reply:
Relic of the original module. I'll do a naming scrub

Reply:
Ya... I think the original had copy/pasta because the postgres module was likely used as a template for this.